index.html

```diff
@@ -0,0 +1,40 @@
+    <link rel="stylesheet" href="style.css">
+    <title>Tribute Page</title>
+</head>
+<body>
+    <div id="main">
+        <h1 id="title">Dr. Norman Borlaug</h1>
+        <p class="important"+>Tribute to the great man</p>
+        <div id="img-div">
+            <img src="img/norman-borlaug.png" alt="" id="image">
+            <div id="img-caption">
+                <p><i>Dr Norman Borlaug in a corn field</i></p>
+            </div>
+        </div>
+        <div id="tribute-link">
+            <h2>Things that we should know about Dr Borlaug:</h2>
+
+            <ul>
+                <li>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Velit pariatur inventore, veritatis dignissimos quis omnis necessitatibus explicabo tempora assumenda quos. Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quod nulla aut magnam, dolores voluptates enim recusandae illum? Magnam exercitationem vel provident, quis aspernatur deleniti delectus.</li>
+                <li>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Velit pariatur inventore, veritatis dignissimos quis omnis necessitatibus explicabo tempora assumenda quos. Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quod nulla aut magnam, dolores voluptates enim recusandae illum? Magnam exercitationem vel provident, quis aspernatur deleniti delectus.</li>
+                <li>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Velit pariatur inventore, veritatis dignissimos quis omnis necessitatibus explicabo tempora assumenda quos. Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quod nulla aut magnam, dolores voluptates enim recusandae illum? Magnam exercitationem vel provident, quis aspernatur deleniti delectus.</li>
+                <li>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Velit pariatur inventore, veritatis dignissimos quis omnis necessitatibus explicabo tempora assumenda quos. Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quod nulla aut magnam, dolores voluptates enim recusandae illum? Magnam exercitationem vel provident, quis aspernatur deleniti delectus.</li>
+                <li>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Velit pariatur inventore, veritatis dignissimos quis omnis necessitatibus explicabo tempora assumenda quos. Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quod nulla aut magnam, dolores voluptates enim recusandae illum? Magnam exercitationem vel provident, quis aspernatur deleniti delectus.</li>
+                <li>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Velit pariatur inventore, veritatis dignissimos quis omnis necessitatibus explicabo tempora assumenda quos. Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quod nulla aut magnam, dolores voluptates enim recusandae illum? Magnam exercitationem vel provident, quis aspernatur deleniti delectus.</li>
+                
+            </ul>
+
+
+        </div>
```

Comment:
Use the semantic tag for the article instead of using div tags.

Reply:
I think it will be better to write in polite suggestion instead of order. In this comment, you tried to explain the solution, but it will be better to use "why" with your version of the solution. 🤓🤓